src/game.cpp

```diff
@@ -1429,16 +1419,16 @@ class Game {
 
 	/* GUI stuff
 	 */
-	gui::IGUIStaticText *guitext;          // First line of debug text
+	gui::IGUIStaticText *guitext;          // First line of status/debug text
 	gui::IGUIStaticText *guitext2;         // Second line of debug text
 	gui::IGUIStaticText *guitext3;         // Third line of debug text
-	gui::IGUIStaticText *guitext_info;     // At the middle of the screen
-	gui::IGUIStaticText *guitext_status;
-	gui::IGUIStaticText *guitext_chat;	   // Chat text
+	gui::IGUIStaticText *guitext_info;     // Object info (e.g. "Furnace out of fuel")
+	gui::IGUIStaticText *guitext_notice;   // GUI messages (e.g. "Minimap hidden")
```

Comment:
The old comment was downright misleading, guitext_info isn't the one "at the middle of the screen", it's at the left. It did take me a while to understand which text does what, so I wanted to make it easier for the next guy...

BTW, I split my changes in 4 separate patches, they're probably easier to read separately.